Makefile

```
@@ -0,0 +1,178 @@
+.PHONY: help build run test clean dev deps lint format check security install-tools setup-hooks pre-commit
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)